t/lib/t/MusicBrainz/Server/Controller/WS/2/SubmitRecording.pm

```diff
@@ -62,6 +62,9 @@ my $content = '<?xml version="1.0" encoding="UTF-8"?>
       </isrc-list>
     </recording>
   </recording-list>
+  <edit-note>
+    Testy test!
+  </edit-note>
```

Comment:
Could we add some newlines here to simulate paragraphs, and some XML entity references (e.g. `&quot;`, `&amp;`) to make sure those are preserved/converted properly?